testutil/test_helper.go

```diff
@@ -366,6 +369,33 @@ func (h *TestHelper) HTTPGetURL(url string) (string, error) {
 	return body, err
 }
 
+// RunLogsAndEventTests runs the TestLogs and TestEvents tests in a separate
+// process
+func (h *TestHelper) RunLogsAndEventTests() {
+	// don't run for tests that were singled out
+	if h.run != "" {
```

Comment:
I'm confused about this `run` field.  It seems that we only check if it's empty but never check the value.  What is it supposed to do?

Reply:
That was used to identify when we shell out, so to avoid calling `TestLogs` and `TestEvents` recursively in the child process, but we're already avoiding that with the `fetchLogsEvents` var, so I think I can get rid of this...